CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # NEXT RELEASE
 
 ### Enhancements
-* <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
-* None.
+* Allow multiple app instances with the same ID but different base URLs ([#6480](https://github.com/realm/realm-core/pull/6480))
```

Comment:
I'd add something here about supporting multiple app instances via get_shared_app() and get_cached_app() specifically - the changelog is used by the SDKs to know when/what to change when they're adopting a version of core and to communicate to users what they'll need to change.

Reply:
Good call, will update 👍🏼